test/usecases/retrieveHospitalWardVisitTotals.js

```diff
@@ -0,0 +1,45 @@
+export default ({ getDb }) => async (hospitalId) => {
```

Comment:
I presume this was moved to the wrong place?

Reply:
Correct, just removed it